app/controllers/clearance/passwords_controller.rb

```diff
@@ -85,8 +86,10 @@ def find_user_by_id_and_confirmation_token
   end
 
   def find_user_for_create
+    password_params = params[:password] || {}
```

Comment:
How would you feel about making this a method to use in both places? Maybe `password_params` or `email_from_params`?